homepage/homepage/app/docs/[...slug]/schemas/accounts.mdx

```diff
@@ -0,0 +1,68 @@
+import { CodeGroup } from "@/components/forMdx";
+
+# Accounts & Migrations
```

Comment:
@bensleveritt can we use sentence case please? Right now we have a mix of sentence case and title case but I'll fix it